doc/rados/operations/placement-groups.rst

```diff
@@ -700,134 +705,146 @@ For example:
+the following:
 
 .. prompt:: bash #
 
    osdmap e13 pg 1.6c (1.6c) -> up [1,0] acting [1,0]
 
 
-Get a PGs Statistics
-====================
+Get a PG's Statistics
+=====================
 
-To retrieve statistics for a particular placement group, execute the following:
+To see statistics for a particular PG, run a command of the following form:
 
 .. prompt:: bash #
 
    ceph pg {pg-id} query
 
 
-Scrub a Placement Group
-=======================
+Scrub a PG
+==========
 
-To scrub a placement group, execute the following:
+To scrub a PG, run a command of the following form:
 
 .. prompt:: bash #
 
    ceph pg scrub {pg-id}
 
-Ceph checks the primary and any replica nodes, generates a catalog of all objects
-in the placement group and compares them to ensure that no objects are missing
-or mismatched, and their contents are consistent.  Assuming the replicas all
-match, a final semantic sweep ensures that all of the snapshot-related object
-metadata is consistent. Errors are reported via logs.
+Ceph checks the primary OSD and any replica nodes, generates a catalog of all
+objects in the PG, and compares the objects against each other in order to
+ensure that no objects are missing or mismatched and that their contents are
+consistent. If the replicas all match, then a final semantic sweep takes place
+to ensure that all of the snapshot-related object metadata is consistent.
+Errors are reported in logs.
 
-To scrub all placement groups from a specific pool, execute the following:
+To scrub all PGs from a specific pool, run a command of the following form:
 
 .. prompt:: bash #
 
    ceph osd pool scrub {pool-name}
 
-Prioritize backfill/recovery of a Placement Group(s)
-====================================================
 
-You may run into a situation where a bunch of placement groups will require
-recovery and/or backfill, and some particular groups hold data more important
-than others (for example, those PGs may hold data for images used by running
-machines and other PGs may be used by inactive machines/less relevant data).
-In that case, you may want to prioritize recovery of those groups so
-performance and/or availability of data stored on those groups is restored
-earlier. To do this (mark particular placement group(s) as prioritized during
-backfill or recovery), execute the following:
+Prioritize backfill/recovery of PG(s)
+=====================================
+
+You might encounter a situation in which multiple PGs require recovery or
+backfill, but the data in some PGs is more important than the data in others
+(for example, some PGs hold data for images that are used by running machines
+and other PGs are used by inactive machines and hold data that is less
+relevant). In that case, you might want to prioritize recovery or backfill of
+the PGs with especially important data so that the performance of the cluster
+and the availability of that data are restored earlier. To mark specific PG(s)
```

Comment:
of their data
s/mark/designate/
(marginal)

s/earlier/sooner/

Reply:
Accepted,
accepted,

and

accepted.